liquidctl/driver/kraken3.py

```diff
@@ -631,6 +631,7 @@ def __init__(
         # 0 = Normal, 1 = +90 degrees, 2 = 180 degrees, 3 = -90(270) degrees
         self.orientation = 0
         self.brightness = 50  # default 50%
+        self.fw = None
```

Comment:
`self._fw` is already set to None in KrakenX3, and was renamed to that in https://github.com/liquidctl/liquidctl/commit/255821125c07eabfdfde71e14b7ea0d7e629dd7d. Please rebase this PR, as I did that a few days after you here.